taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -343,6 +330,32 @@ def train_action(parameters, graph_config, input, task_group_id, task_id):
 
     parameters = dict(parameters)
 
+    start_stage = input.pop("start-stage", None)
```

Comment:
Why `pop`? Why not `get` and default to `None` like `previous_group_ids`?

Reply:
Good callout. The `get` for `previous_group_ids` is actually a copy/paste error from https://github.com/mozilla-releng/mozilla-taskgraph/blob/cb7a5d6440acc1cfad8f89ec226cc53d6c61f7ba/src/mozilla_taskgraph/actions/release_promotion.py now that I look at it. (Over in that code we don't put `input` into parameters, so it doesn't matter...but over here we do, so we ought to remove it.)